src/m365/purview/commands/retentionevent/retentionevent-remove.spec.ts

```diff
@@ -141,12 +141,12 @@ describe(commands.RETENTIONEVENT_REMOVE, () => {
   });
 
   it('Correctly deletes retention event by id when prompt confirmed', async () => {
-    sinon.stub(request, 'delete').callsFake((opts) => {
-      if (opts.url === `https://graph.microsoft.com/v1.0/security/triggers/retentionEvents/${validId}`) {
-        return Promise.resolve();
+    sinon.stub(request, 'delete').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/security/triggers/retentionEvents/${validId}`) {
```

Comment:
It's odd that GitHub thinks you changed it to beta while this is already like this in the `main` branch 🤔

Reply:
jup, it's pretty odd, maybe something went wrong with `rebase main`